### PR TITLE
Update INSTALL.md #165

### PR DESCRIPTION
**Pull request by richarddushime**
On cloning, installing the requirements and running the Saskatoon-ng repo some errors occured
#165
to solve this issue  i updated even  env.template file by adding the export keyword  the  variables

### CODE REVIEW

**Review by tristanlatr — APPROVED** [review]
Hi @richarddushime,

Glad to see that your issue got resolved. 

I feel like this change should not be necessary for django-dotenv to read your config file.
But since [it supports adding the export keyword](https://github.com/jpadilla/django-dotenv/blob/c68fd4bf3ae330c7160a620210f760f0a52359fd/tests.py#L68), I don't see any reason why not adding it.

File: INSTALL.md [hunk 1]
```diff
@@ -115,11 +123,11 @@ $ sudo mysql -u root -p
 Example mysql configuration in *.env* file:
```
Comment (2ynn) [inline INSTALL.md:123]: @richarddushime here we specify the content goes in the `.env` file (to be parsed by `django-dotenv`)

**Comment by 2ynn** [comment]
@richarddushime same i'm gonna close this one, let me know if you still have issues importing environment variables by adapting the current `.env` template file.

**Comment by 2ynn** [comment]
@richarddushime  @tristanlatr sorry just saw the comment about `django-dotenv` supporting the `export` keyword. My bad!
I don't think it's necessary though. Let me know how you feel about it.

**Comment by richarddushime** [comment]
> @richarddushime @tristanlatr sorry just saw the comment about `django-dotenv` supporting the `export` keyword. My bad! I don't think it's necessary though. Let me know how you feel about it.

on my side i faced several errors while setting up the project and i tried to fixed the issue in several ways  still  could read  the .env  file variables  at the end i got a  quick solution for the problem  by using the ``` export ```keyword in the file though there is([supports adding the export keyword](https://github.com/jpadilla/django-dotenv/blob/c68fd4bf3ae330c7160a620210f760f0a52359fd/tests.py#L68),)  Actually i also guided someone  who was facing the same issue   and it worked well with export keyword in .env file

**Review by tristanlatr — CHANGES_REQUESTED** [review]
Yeah, 

I'm not sure we should introduce these changes, I kinda agree with @2ynn  on that. 
I think the issue resides in the location of the `.env` file being for clear enough, but I'm not certain.

Sorry for the trouble setting up saskatoon @richarddushime.

The little docs additions in the `INSTALL.md` looks good nonetheless.

**Comment by 2ynn** [comment]
Yea I agree the sentence "Copy the `saskatoon/env.template` file into `saskatoon/.env` and adapt it to your needs." could be improved.
What I meant by this is that the `.env` file must be copied at the same location as the template file, i.e.:
```
cp saskatoon/env.template saskatoon/.env
```
And "to your needs" could for sure be improved, (e.g. detailing that the secret key must be added to the file after being generated etc...)

**Pull request closed without merging**